Makefile

```diff
@@ -97,7 +97,7 @@ volume-manager-image:
 	@echo "----------------------------"
 	@PNAME=${VOLUME_MANAGER} CTLNAME=${VOLUME_MANAGER} sh -c "'$(PWD)/build/build.sh'"
 	@cp bin/${VOLUME_MANAGER}/${VOLUME_MANAGER} build/volume-manager/
-	@cd build/${VOLUME_MANAGER} && sudo docker build -t ${HUB_USER}/${VOLUME_MANAGER}:${IMAGE_TAG} --build-arg BUILD_DATE=${BUILD_DATE} .
+	@cd build/${VOLUME_MANAGER} && sudo docker build -t ${HUB_USER}/cstor-${VOLUME_MANAGER}:${IMAGE_TAG} --build-arg BUILD_DATE=${BUILD_DATE} .
```

Comment:
Better to update the VOLUME_MANAGER with cstor-volume-manager

Reply:
The VOLUME_MANAGER variable is the directory name and binary name, IMO it should not be changed to cstor-volume-manager as the parent directory has cstor in it. We can a have a separate variable VOLUME_MANAGER_IMAGE as cstor-volume-manager.